pubsublite/pscompat/subscriber.go

```diff
@@ -75,6 +75,22 @@ func (ah *pslAckHandler) OnNack() {
 	ah.subInstance = nil
 }
 
+// OnAckWithResult is required implementation for the ack handler
+// for Cloud Pub/Sub's exactly once delivery feature. This will
+// ack the message and return nil.
+func (ah *pslAckHandler) OnAckWithResult() *pubsub.AckResult {
+	ah.OnAck()
+	return nil
```

Comment:
There seems to be a discrepancy between the comments in internal/pubsub/message.go ("if using Pub/Sub Lite, AckResult readies immediately with a AcknowledgeStatus.Success") and the return value here.

Should this return a non-nil `pubsub.AckResult`? Same below for `OnNackWithResult`.

Please correct me, if I've misread.

Reply:
Hm yeah in hindsight we should probably just return an AckResult that returns success instead. I'll update this and the comments.

Reply:
LGTM for pubsublite. Thanks!